app/components/SignUp/SubmissionModal.js

```diff
@@ -101,5 +101,4 @@ SubmissionModal.propTypes = {
   t: PropTypes.func.isRequired
 };
 
-
 export default connect(mapStateToProps, null)(withTranslation()(SubmissionModal));
```

Comment:
Above in mapStateToProps it looks like `values` is not being used.

And:
```suggestion
export default connect(mapStateToProps)(withTranslation()(SubmissionModal));
```